apps/web/src/app/domain/pages/themes/components/theme-preview/theme-preview.component.html

```
@@ -291,15 +344,20 @@ <h3 class="text-foreground text-lg font-semibold">Dashboard Preview</h3>
                     </div>
                   }
                 </div>
-                <z-separator class="my-8" />
-                <button z-button zType="outline" class="w-full" zSize="sm">
-                  <ng-icon name="lucidePlus" class="mr-2 size-4!" />
-                  Invite Member
-                </button>
+                <z-card-footer zFooterBorder>
+                  <button z-button zType="outline" class="w-full" zSize="sm">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)